### PR TITLE
Updates use of component and unit display names

### PR DESCRIPTION
**Pull request by lamagnifica**
## [DOC-2913](https://openedx.atlassian.net/browse/DOC-2913)

Component names are no longer listed when you mouseover the learning sequence. This PR tunes up descriptions of why you give everything a good display name. I also found some out of date images, particularly for bookmarks.
### Date Needed

This feature change went live in April.
### Reviewers

Possible roles follow. PR submitter checks the boxes after each reviewer finishes and gives :+1:. 
- [x] Subject matter expert: @clrux
- [ ] Subject matter expert: 
- [x] Doc team review (sanity check/copy edit/dev edit): @pdesjardins @catong @srpearce 
- [ ] Product review:
- [ ] Partner support: 
- [ ] PM review: 

FYI: @mmacfarlane @jaakana @mattdubose
### Testing
- [x] Ran ./run_tests.sh without warnings or errors
### HTML Version (optional)
- [ ] Several guides were affected, but most of the changes can be seen in context with screen shots in the B&R guide: http://draft-display-names-2016-05-14.readthedocs.io/en/latest/ specific links with the files.

Let me know if you need an HTML version or screen shot of any of the other affected guides.
### Post-review
- [ ] Add description to release notes task as a comment
- [x] Squash commits

### CODE REVIEW

File: en_us/shared/course_components/create_discussion.rst [hunk 1]
```diff
@@ -14,8 +14,9 @@ This section describes how to work with discussion components in Studio.
 Overview
```
Comment (lamagnifica) [inline en_us/shared/course_components/create_discussion.rst:14]: See http://draft-display-names-2016-05-14.readthedocs.io/en/latest/course_components/create_discussion.html#a-learner-s-view-of-the-discussion

File: en_us/shared/course_components/create_problem.rst [hunk 2]
```diff
@@ -310,19 +310,23 @@ select **Settings** in the component editor.
 Display Name
 ===============
```
Comment (lamagnifica) [inline en_us/shared/course_components/create_problem.rst:205]: see http://draft-display-names-2016-05-14.readthedocs.io/en/latest/course_components/create_problem.html#display-name

File: en_us/shared/developing_course/course_components.rst [hunk 3]
```diff
@@ -24,24 +24,18 @@ What is a Component?
 A component is the part of a unit that contains your actual course content. A
```
Comment (lamagnifica) [inline en_us/shared/developing_course/course_components.rst:24]: see http://draft-display-names-2016-05-14.readthedocs.io/en/latest/developing_course/course_components.html

File: en_us/shared/developing_course/course_outline.rst [hunk 4]
```diff
@@ -76,16 +76,18 @@ your course content.
 .. image:: ../../../shared/images/Course_Outline_LMS.png
```
Comment (lamagnifica) [inline en_us/shared/developing_course/course_outline.rst:76]: see http://draft-display-names-2016-05-14.readthedocs.io/en/latest/developing_course/course_outline.html#view-the-course-organization-as-a-learner

File: en_us/shared/developing_course/course_units.rst [hunk 5]
```diff
@@ -68,16 +68,17 @@ Viewing Units as a Learner
 ****************************
```
Comment (lamagnifica) [inline en_us/shared/developing_course/course_units.rst:68]: see http://draft-display-names-2016-05-14.readthedocs.io/en/latest/developing_course/course_units.html#viewing-units-as-a-learner

File: en_us/shared/developing_course/course_outline.rst [hunk 6]
```diff
@@ -76,16 +76,18 @@ your course content.
...
-   see the subsections and units that they contain, or collapse for a compact
-   overview.
-#. Subsections and units appear in the left pane when learners expand a
-   section.
-#. Components in the selected unit appear in the learning sequence. Learners
-   select icons in the learning sequence to access specific components in the
-   current unit. They can also use the left and right arrow buttons at either
-   end of the learning sequence to move back to the previous unit or forward to
-   the next unit.
+#. Sections are listed in the left pane. Learners select a section to see the
```
Comment (downzer0) [inline en_us/shared/developing_course/course_outline.rst]: Do we normally use "left"/"right" directionals like this? What about for RTL users?

**Comment by downzer0** [comment]
Had one question, but otherwise looks good to me.

File: en_us/shared/developing_course/course_outline.rst (hunk 6 above)
Comment (lamagnifica) [inline en_us/shared/developing_course/course_outline.rst]: @clrux yes, it's been a source of some discussion. Do you have a suggestion? "Side navigation pane"? "side pane"? It used to be called the "course accordion" but that wasn't meaningful to anyone. Or, maybe, duh, what does the DOM call it?

Comment (downzer0) [inline en_us/shared/developing_course/course_outline.rst]: We call it the "Course Navigation" in the DOM, for screenreaders.

Comment (lamagnifica) [inline en_us/shared/developing_course/course_outline.rst]: hunh
I just looked and it reads "aria-label="Course Material"
The horizontal nave bar above the course content, where you choose the pages, is called "Course View", which I also did not know...

**Comment by lamagnifica** [comment]
@clrux I updated all files in this PR to use the terminology we discussed for course navigation and unit navigation. @srpearce @catong @pdesjardins this commit actually gives us a head start on the glossary change and some other changes we talked about today.

**Comment by downzer0** [comment]
👍

**Comment by lamagnifica** [comment]
@pdesjardins @catong or @srpearce please review when you have a moment

File: en_us/olx/source/problem-xml/create_problem.rst [hunk 7]
```diff
@@ -87,18 +87,19 @@ All problems on the edX platform have several component parts.
...
-    the learner sees in the problem.
-  - If the number of Maximum Attempts that was set for this problem has been
-    reached, the **Reset** button is not visible.
+   * If the learner has already submitted an answer, selecting **Reset** clears
+     the submission and, if the problem includes a Python script to randomize
+     variables and the randomization setting is **On Reset**, changes the
+     values the learner sees in the problem.
 
-12. **Hide Answer button.**
+   * If the number of Maximum Attempts that was set for this problem has been
```
Comment (pdesjardins) [inline en_us/olx/source/problem-xml/create_problem.rst]: Lower case "maximum attempts." Or is this a UI label that needs bold formatting?

File: en_us/shared/course_components/create_discussion.rst [hunk 8]
```diff
@@ -14,8 +14,9 @@ This section describes how to work with discussion components in Studio.
...
 *******************
 
-You can add discussion components to a unit to pose questions related to the
-unit and give learners a chance to respond and interact on a specific subject.
+You can add discussion components to a unit to pose questions about other
+components, such as a video or text, in the unit. A discussion component gives
```
Comment (pdesjardins) [inline en_us/shared/course_components/create_discussion.rst]: about other components, such as a videos or texts, in the unit

File: en_us/shared/course_components/create_discussion.rst [hunk 9]
```diff
@@ -14,8 +14,9 @@ This section describes how to work with discussion components in Studio.
...
+learners a chance to respond and interact on a specific subject.
```
Comment (pdesjardins) [inline en_us/shared/course_components/create_discussion.rst]: > respond and interact on a specific subject

respond to and interact with each other about a specific subject

File: en_us/shared/course_components/create_problem.rst [hunk 10]
```diff
@@ -596,13 +600,12 @@ Modifying a Released Problem
...
-After a learner submits a response to a problem, the edX Learning Management
-System (LMS) stores that response, the score that the learner
-received, and the maximum score for the problem. For problems with a **Maximum
-Attempts** setting greater than 1, the LMS updates these values each time the
-learner submits a new response to a problem. However, if you change
-a problem or its attributes, existing learner information for that problem is
-not automatically updated.
+After a learner submits a response to a problem, the LMS stores that response,
+the score that the learner received, and the maximum score for the problem. For
+problems with a **Maximum Attempts** setting greater than 1, the LMS updates
```
Comment (pdesjardins) [inline en_us/shared/course_components/create_problem.rst]: 1 > one

File: en_us/shared/developing_course/course_components.rst [hunk 11]
```diff
@@ -118,13 +112,13 @@ To set the display name for a component, follow these steps.
 #. Edit the **Display Name** field.
 
   .. image:: ../../../shared/images/display-name.png
-   :alt: The settings dialog box for an HTML component.
-   :width: 600
+   :alt: The settings dialog box for an problem component.
```
Comment (pdesjardins) [inline en_us/shared/developing_course/course_components.rst]: an problem > a problem

File: en_us/xblock-tutorial/source/edx_platform/edx_lms.rst [hunk 12]
```diff
@@ -33,16 +33,26 @@ The LMS requires XBlocks to have the following properties.
...
-  displayed to learners in the learning sequence header in courseware when the
-  XBlock is in that unit.  The variable must have one of the following values.
-
-  * ``problem``
-  * ``video``
-  * ``other``
+.. * A method named ``get_progress`` that . . . TBD (  x_module.py:XModuleMixin.get_progress)
+
+* A class property named ``icon_class``, which controls the icon that
+  displays to learners in the learning sequence on the **Course** page when the
```
Comment (pdesjardins) [inline en_us/xblock-tutorial/source/edx_platform/edx_lms.rst]: learning sequence > unit navigation bar ?

**Comment by pdesjardins** [comment]
Great! 👍

File: en_us/shared/course_components/create_discussion.rst (hunk 8 above)
Comment (lamagnifica) [inline en_us/shared/course_components/create_discussion.rst]: @pdesjardins I don't understand. strike "a"?

File: en_us/shared/course_components/create_problem.rst (hunk 10 above)
Comment (lamagnifica) [inline en_us/shared/course_components/create_problem.rst]: @pdesjardins even when what you enter in the field is "1"?

File: en_us/shared/course_components/create_discussion.rst (hunk 8 above)
Comment (pdesjardins) [inline en_us/shared/course_components/create_discussion.rst]: Yes, and make videos and texts plural, for number agreement.

File: en_us/shared/course_components/create_problem.rst (hunk 10 above)
Comment (pdesjardins) [inline en_us/shared/course_components/create_problem.rst]: If it's a non-translatable configuration value, then `1`, I think. MMOS is a little waffley about spelling out numbers, I was just reacting to a number less than ten as a numeral. This isn't part of the intent of your PR, so ignore if you prefer.

File: en_us/shared/course_components/create_discussion.rst [hunk 13]
```diff
@@ -85,28 +86,21 @@ Create a Discussion Component
...
-
-.. image:: ../../../shared/images/DiscussionComponent_LMS_Ribbon.png
- :alt: An image of a unit from a learners's point of view, with the component
-  list showing a discussion component.
- :width: 600
-
-The discussion space appears under other components in the unit. It does not
-have a label in the body of the unit. Instead, learners see "Show discussion"
-or "Hide discussion" on the left, and **New Post** on the right.
+For learners, a content-specific discussion topic appears under the other
```
Comment (catong) [inline en_us/shared/course_components/create_discussion.rst]: "appears under"? It's not clear which physical area you're referring to.

Comment (catong) [inline en_us/shared/course_components/create_discussion.rst]: I'd make plural: "For learners, content-specific discussion topics appear..."

Comment (lamagnifica) [inline en_us/shared/course_components/create_discussion.rst]: good point, @catong and I realized that the first sentence didn't make any sense, before or after my change... the discussion component appears in whatever order you add it in the unit, not always at the bottom...

**Pull request merged**